common/test/unit/com/thoughtworks/go/util/OperatingSystemTest.java

```diff
@@ -12,40 +12,20 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *************************GO-LICENSE-END***********************************/
+ */
 
 package com.thoughtworks.go.util;
 
+import org.junit.Test;
+
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
 
-import com.thoughtworks.go.util.OperatingSystem;
-import org.junit.Test;
-
 public class OperatingSystemTest {
```

Comment:
This file is not necessary to be kept, is it? Now that there are no tests in it.